CHANGELOG.md

```diff
@@ -1,4 +1,201 @@
-### [Versions](https://material-ui.com/versions/)
+### [Versions](https://material-ui.com/versions/
+
+## 5.0.0-alpha.38
+
+<!-- generated comparing v5.0.0-alpha.37..next -->
+
+_Jun 23, 2021_
+
+Big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 We have completed all the [breaking changes](https://github.com/mui-org/material-ui/issues/20012). The plan to release 5.0.0-beta.0 is on July 1st and will start to promote its usage over v4
```

Comment:
```suggestion
- 🚀 We have completed all the [breaking changes](https://github.com/mui-org/material-ui/issues/20012). The plan to release 5.0.0-beta.0 is on July 1st and will start to promote its usage over v4.
```

Reply:
But I'm not so sure about this. For instance:

- We have removed the span from Button and IconButton, but didn't do it from BottomNavigationAction. Shouldn't we do the same?
-

Reply:
Agree, I will add them to the list and update the changelog